hypergan/trainers/joint_trainer.py

```diff
@@ -105,14 +105,9 @@ def run(gan, feed_dict):
 
     _, d_cost, d_log = sess.run([d_optimizer, d_loss, d_log_t])
 
-    if(d_class_loss is not None):
-        _, g_cost,k, measure = sess.run([g_optimizer, g_loss,gan.graph.update_k], feed_dict)
-        if iteration % 100 == 0:
-            print("%2d: g cost %.2f d_loss %.2f %.6f k %.2f m" % (iteration,g_cost,  d_cost,g_k, measure))
-    else:
-        _, g_cost, g_k, measure= sess.run([g_optimizer, g_loss, gan.graph.update_k, gan.graph.measure], feed_dict)
-        if iteration % 100 == 0:
-            print("%2d: g cost %.2f d_loss %.2f  %.6f k %.2f m" % (iteration,g_cost , d_cost,g_k, measure))
+    _, g_cost, g_k, measure= sess.run([g_optimizer, g_loss, gan.graph.update_k, gan.graph.measure], feed_dict)
```

Comment:
This now alternates, so maybe the name `joint_trainer` is incorrect.  I'm not sure the correct name, but it trains gan.graph.update_k which is defined in `began` from `Proportional Control Theory`.

Maybe `propotional_control_trainer`?